.pre-commit-config.yaml

```diff
@@ -7,34 +7,28 @@
 #
 # - Register git hooks: pre-commit install --install-hooks
 # - Run on all files:   pre-commit run --all-files
+ci:
+  autoupdate_schedule: monthly
+  autofix_commit_msg: |
+    [pre-commit.ci] Apply automatic pre-commit fixes
 
 repos:
   - repo: https://github.com/psf/black
-    rev: 23.9.1
+    rev: 24.3.0
     hooks:
       - id: black
         exclude: "examples|tests/assets"
 
   - repo: https://github.com/charliermarsh/ruff-pre-commit
     # Ruff version.
-    rev: "v0.0.289"
+    rev: "v0.3.4"
     hooks:
       - id: ruff
         exclude: "examples|tests/assets"
         args: ["--fix"]
 
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-      - id: isort
-        name: isort
-        additional_dependencies: [toml]
-        files: \.py$
-        args: ["--profile", "black"]
-        exclude: "conda-store-server/conda_store_server/action/__init__.py"
-
```

Comment:
Why is this removed? Is it assumed that pre-commit will handle this?

I reordered some imports in `conda-store-server/conda_store_server/api.py`, but `hatch env run -e lint lint` didn't fix that.

Reply:
I had swapped to the `isort` in `ruff` but it makes little difference to be honest and if it makes it easier/less confusing to keep the sort pre-commit it is ok